src/Icons/src/Iconify.php

```diff
@@ -85,6 +85,52 @@ public function fetchIcon(string $prefix, string $name): Icon
         ]);
     }
 
+    public function fetchIcons(string $prefix, array $names): array
+    {
+        if (!isset($this->sets()[$prefix])) {
+            throw new IconNotFoundException(\sprintf('The icon set "%s" does not exist on iconify.design.', $prefix));
+        }
+
+        // Sort to enforce cache hits
+        $queryString = implode(',', $names);
```

Comment:
the comment above says "sort" (to get more cache hits) but here you are not sorting the values, just imploding them. The sorting of the `$names` array is missing, right?